doc/man3/EVP_EncryptInit.pod

```diff
@@ -251,6 +255,12 @@ EVP_CipherFinal_ex(). In previous releases they also cleaned up
 the B<ctx>, but this is no longer done and EVP_CIPHER_CTX_clean()
 must be called to free any context resources.
 
+EVP_Cipher() encrypts or decrypts as many EVP_CIPHER_block_size() block from
+I<in> that I<inl> allows and places the result in I<out>.  It returns the
+number of encrypted or decrypted bytes.  This function does not buffer anything
+like EVP_CipherUpdate does, and therefore gives a very direct access to the
+central block cipher of the cipher implementation.
```

Comment:
Maybe something like.. (note its also used for streaming ciphers - so the block cipher comment is a bit misleading?)

This function gives direct access to the low level cipher implementation. If buffering is required EVP_CipherUpdate should be used instead.

Reply:
Okie

Reply:
I did a little more rephrasing.